pkg/feeds/crates/crates.go

```diff
@@ -41,7 +41,15 @@ func fetchPackages(baseURL string) ([]*Package, error) {
 	if err != nil {
 		return nil, err
 	}
-	resp, err := httpClient.Get(pkgURL)
+	req, err := http.NewRequest("GET", pkgURL, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	// set headers
+	req.Header.Set("Accept", "application/json")
+	req.Header.Set("User-Agent", "package-feeds")
```

Comment:
Can we make the user agent `package-feeds (github.com/ossf/package-feeds)`?

Having the repository in the header gives administrators a point of contact if needed.

I have another change I'm preparing that adds this user-agent to all feeds, but I'm happy to proceed with this change to unblock you.